grails-app/controllers/au/org/ala/volunteer/UserController.groovy

```diff
@@ -618,40 +621,75 @@ class UserController {
         log.debug("ajaxGetPoints| Transcription.countByFullyTranscribedBy(): ${sw.toString()}")
         sw.reset().start()
 
-        final query = """{
-  "constant_score": {
-    "filter": {
-      "and": [
-        { "term": { "transcriptions.fullyTranscribedBy": "${userInstance.userId}" } },
-        { "nested" :
-          {
-            "path" : "fields",
-            "filter" : { "term" : { "name": "decimalLongitude"}}
-          }
-        },
-        { "nested" :
-          {
-            "path" : "fields",
-            "filter" : { "term" : { "name": "decimalLongitude"}}
-          }
+        final query = """
+{
+    "query": {
+        "bool": {
+            "must": [
+                {
+                    "term": { 
+                        "transcriptions.fullyTranscribedBy": "${userInstance.userId}" 
+                    }
+                },
+                {
+                    "term": { 
+                        "fields.name": "decimalLongitude" 
+                    }
+                },
+                {
+                    "term": { 
+                        "fields.name": "decimalLatitude" 
+                    }
+                }
+            ]
         }
-      ]
     }
-  }
 }"""
-
-        def searchResponse = fullTextIndexService.rawSearch(query, SearchType.QUERY_THEN_FETCH, taskCount.intValue(), fullTextIndexService.rawResponse)
+        // Elastic Search max value.
+        final int MAX_SEARCH = 10000
+        def maxSearchResults = (MAX_SEARCH <= taskCount.intValue() ? MAX_SEARCH : taskCount.intValue())
```

Comment:
Given it doesn't look like you are paging here, could you just pass in MAZ_SEARCH as your max rather than doing the extra calculation?